### ISSUE TITLE
Error when capturing on windows with compression problem

### ISSUE THREAD

**Issue by uehaj2022** [uehaj2022/latteart#3]
**Describe the bug**
With using latteart catpture tool, start recording, get Error dialog, and typescript error about compression.

**To Reproduce**
1. Go to capture tool
2. Enter URL (eg. https://www.google.com/)
3. Click recording button.
4. Error dialog  "An error has occurred" opened.
5. See typescript stacktrace error (on chrome devtool)

**Expected behavior**
No error

**Screenshots**
<img width="1227" alt="image" src="https://user-images.githubusercontent.com/112670104/192929135-50ac7dc7-e143-4175-8af9-bfbf0a621e4f.png">

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome (maybe)
 - Version Unknown

**Additional context**
-

**Comment by GochaLearningSpace** [comment]
> **Describe the bug** With using latteart catpture tool, start recording, get Error dialog, and typescript error about compression.
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. Go to capture tool
> 2. Enter URL (eg. https://www.google.com/)
> 3. Click recording button.
> 4. Error dialog  "An error has occurred" opened.
> 5. See typescript stacktrace error (on chrome devtool)
> 
> **Expected behavior** No error
> 
> **Screenshots** <img alt="image" width="1227" src="https://user-images.githubusercontent.com/112670104/192929135-50ac7dc7-e143-4175-8af9-bfbf0a621e4f.png">
> 
> **Desktop (please complete the following information):**
> 
> * OS: Windows
> * Browser Chrome (maybe)
> * Version Unknown
> 
> ## **Additional context**

@uehaj2022 
I try to fill information in below, but if you have preferred procedures, please let me know.

* Error message.
--
An error has occurred.
An unknown error occurs. 
--

*Windows Version 
--
Edition	Windows 11 Pro
Version	21H2
Installed on	‎2022/‎08/‎17
OS build	22000.978
Experience	Windows Feature Experience Pack 1000.22000.978.0
--
![image](https://user-images.githubusercontent.com/6176526/193372244-cca7b6a1-a1a3-4182-b782-a813e8a427a6.png)

*Chrome Version
--
Google Chrome
Nearly up to date! Relaunch Chrome to finish updating.
Version 106.0.5249.62 (Official Build) (64-bit)
--
![image](https://user-images.githubusercontent.com/6176526/193372566-1fe96bfc-95b4-4dc8-a19e-4caa964564a2.png)

*Version of Latteart
@uehaj2022 
Where I should be able to confirm the version of Latteart ?  (v1.7.5 is the one ? )
![image](https://user-images.githubusercontent.com/6176526/193374182-4363696e-9ec3-47fa-b2e9-b4eb7b732320.png)

**Comment by dikshika-s** [comment]
Respected Sir
I am also facing similar issue with capture tool despite having set up the path. I have tried multiple ways like resetting up multiple times, making some changes but in vein.

My windows version is: Windows 10 Version 21H2 (OS Build 19044.2006)
My chrome version is: Version 106.0.5249.103 (Official Build) (64-bit)

Attached below are the screen shots of the issue:
![Screenshot 2022-10-11 221248](https://user-images.githubusercontent.com/79650403/195151293-5423935a-5da1-4762-bb13-16acdd3dc54f.png)
on clicking OK, Again error message appears
![Screenshot 2022-10-11 221317](https://user-images.githubusercontent.com/79650403/195151371-ca0cb8a3-8131-4b55-93d4-f392c48d7a50.png)
On again clicking ok and creating test result then, on executing it displays an error
![Screenshot 2022-10-11 221233](https://user-images.githubusercontent.com/79650403/195151445-b01e5da8-cdae-46ad-9cf0-f27217dfcdda.png)

The same dialog box pops on running the management tool too
![image](https://user-images.githubusercontent.com/79650403/195151567-69ed80a1-9460-48bf-9c79-0c7a8cbb8da5.png)


Kindly help to resolve it
Thank you

**Comment by uehaj2022** [comment]
Hi, dikshika-s san, thanks for report.
Please check the directory of webp listed in above path elements.

 - C:\Users\91869\AppData\Local\Yarn\bin\libwebp-0.4.1-rc1-windows\x64\lib\....
 
is snipped so check whole of the full-path directory, and check the files under the directory.
Can you find cweb.exe in there?

**Comment by dikshika-s** [comment]
Yes Sir the full path is- C:\Users\91869\Downloads\libwebp-0.4.1-rc1-windows-x64\libwebp-0.4.1-rc1-windows-x64\bin . Attached herewith is the directory screenshot.
![image](https://user-images.githubusercontent.com/79650403/195268134-cd177fe1-a45d-4e6d-8a68-d5e58cf068c9.png)

Please guide regarding the same
Thank you

**Comment by uehaj2022** [comment]
OK, Thank you for give me the information.
Gocha-san reported:
https://github.com/uehaj2022/latteart/issues/3#issuecomment-1264148531

You can see:
https://user-images.githubusercontent.com/112670104/192929135-50ac7dc7-e143-4175-8af9-bfbf0a621e4f.png

![image](https://user-images.githubusercontent.com/112670104/195272926-cd905f8a-6c76-413a-bde1-903cb672c727.png)

Right side of this browser screenshot(maybe chrome),  chrome developer tool shown.
And errors and exceptions are observed.
Can you report  chrome devtool output information when the error occurs?

**Comment by dikshika-s** [comment]
Yes Sir.
This is the error being displayed
![image](https://user-images.githubusercontent.com/79650403/195298762-b9681368-caf9-44a5-b2f7-992dbb631371.png)
On clicking ok and trying to create test this was displayed further, 
![image](https://user-images.githubusercontent.com/79650403/195299099-c54f0d90-7df8-4c11-b41a-3307e5412255.png)

**Comment by dikshika-s** [comment]
Thank you Sir, I found the bug and was able to resolve and Run it successfully.
Thank You

**Comment by dikshika-s** [comment]
Respected Sir
Thank you for the support . The bug is resolved, and Latteart is running successfully on my laptop.
Kind Regards

**Comment by uehaj2022** [comment]
Nicely done! Thanks for your effort.
If you think it is valuable that sharing the reason of the error reported, please let me know.
If the reason is not to be shared in some reason, please ignore and please try and do your valuable work!
best regards,

**Comment by dikshika-s** [comment]
Sir the reason was incompatibility of the versions of some dependencies. I resolved it installing another latteart version and syncing .

**Comment by uehaj2022** [comment]
Thank you for the information!
